addon/-private/adapters/build-url-mixin.js

```diff
@@ -47,16 +47,17 @@ export default Ember.Mixin.create({
     @param {String} modelName
     @param {(String|Array|Object)} id single id or array of ids or query
     @param {(DS.Snapshot|Array)} snapshot single snapshot or array of snapshots
+    @param {(DS.SnapshotRecordArray)} snapshotRecord single snapshotRecordArray
```

Comment:
OK, I was confused with this as I wasn't sure if it was an array of Snapshot or the DS.SnapshotRecordArray. Will update...